### PR TITLE
all: fix GetPriorityClass with default

### PR DESCRIPTION
**Pull request by hormes**
### Ⅰ. Describe what this PR does

1. Koordinator is using PriorityClass to make a lot of logical judgments, which need to consider compatible scenarios (default value)
2. Renamed GetPriorityClass and GetQoSClass at the same time to avoid misuse

### CODE REVIEW

File: pkg/descheduler/utils/sorter/pod_test.go [hunk 1]
```diff
@@ -130,7 +130,7 @@ func TestSortPods(t *testing.T) {
 		corev1.ResourcePods:   1,
 	}
 	SortPodsByUsage(pods, podMetrics, nodeAllocatableMap, resourceToWeightMap)
```
Comment (hormes) [inline pkg/descheduler/utils/sorter/pod_test.go:132]: Here you need to focus on it, because it will affect the sorting results. @eahydra

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Review by saintube — COMMENTED** [review]
/lgtm

File: pkg/descheduler/utils/sorter/pod_test.go (hunk 1 above)
Comment (eahydra) [reply pkg/descheduler/utils/sorter/pod_test.go:132]: This new ordering looks reasonable.

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Pull request merged**